core/ckeditor.js

```diff
@@ -33,6 +33,44 @@ CKEDITOR.instances = {};
  */
 CKEDITOR.document = new CKEDITOR.dom.document( document );
 
+/**
+ * Gets the element from DOM and checks if the editor can be instantiated on it.
+ * This function is available for internal use only.
+ *
+ * @private
+ * @since 4.12.0
+ * @returns {CKEDITOR.dom.element/null}
+ */
+CKEDITOR.getEditorElement = function( elementOrId ) {
+	var element;
```

Comment:
You should just declare this `var` together with initialization few lines below:

```
element = CKEDITOR.dom.element.get( elementOrId );
```